4/test/run

```diff
@@ -40,7 +40,12 @@ container_logs() {
 }
 
 run_s2i_build() {
-  s2i build ${s2i_args} file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
+  local NODE_ENV=$1
+  if [ ! $NODE_ENV ]; then
```

Comment:
No need for `if/else` there. If `NODE_ENV` is not set or is empty it will be set to production.

So `s2i build ${s2i_args} -e "NODE_ENV=$NODE_ENV" file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp` should be sufficient.

Reply:
Removed in my latest commit